llm/nemorl/nemorl.sky.yaml

```diff
@@ -48,17 +48,25 @@ run: |
   cd nemo-rl
   head_ip=`echo "$SKYPILOT_NODE_IPS" | head -n1`
   num_nodes=`echo "$SKYPILOT_NODE_IPS" | wc -l`
+  # Isolate inner Ray from outer Ray env
+  unset RAY_RAYLET_PID
```

Comment:
For discussion - should we unset RAY_RAYLET_PID this generally for all `run` and `setup` instead of having it in individual task YAMLs?

Reply:
We should also probably update the [distributed ray example](https://github.com/skypilot-org/skypilot/blob/master/examples/distributed_ray_train/ray_train.yaml) if we decide to keep this in the YAML

Reply:
Will unset the env for all `run` and `setup` phases.